src/current/v23.2/migration-strategy-live-migration.md

```diff
@@ -0,0 +1,76 @@
+- It requires abstracting the database operations in the application. You need complete control over all the applications reading and writing to the database to ensure the data is consistent across both the source and target database.
+- During migration there's less certainty about the integrity of the data due to multiple systems being updated.
+- Hidden dependencies may reveal themselves after cutover. For example, a microservice that was not accounted for during the migration planning and uses the source database may break after cutover.
+- The operational cost of two production systems can be expensive.
+
+## Prepare for a live migration
+
+Before performing a live migration, consider the following:
+
+- It is better to over-provision your cluster during the migration and then scale it back after the migration is complete than to discover you need more cluster resources during the migration.
+- The more testing you do before the migration to properly size your cluster and model the performance of your production workload, the more likely it is that your migration will succeed with minimal problems.
+- Writes are typically where performance problems arise during live migration. Testing your workload beforehand can help identify areas with slow write performance so they can be fixed before starting the migration.
+- After the migration completes, you should factor in time for performance tuning.
+- A common way of testing the performance of your workload on CockroachDB is to do gradual testing while the source database is the source of truth, and the CockroachDB cluster's data is replicated from the source database.
+- Fallback is typically quite expensive, and it is usually easier to fix any data consistency issues after the cutover to the target database than to fallback to the original source database.
+
+## Live migration tooling
+
+Live migration requires tooling to copy the data from the source database to the target database, and synchronize the data back to the source database after cutover. If you need to import an initial dataset, you can choose to perform the import before the migration, or concurrent with the live migration.
+
+The initial data migration and synchronization can be handled by tools such as:
+
+- [AWS Database Migration Service]({% link {{ page.version.version }}/aws-dms.md %}) (DMS)
+- [Qlik Replicate]({% link {{ page.version.version }}/qlik.md %})
+- [Striim({% link {{ page.version.version }}/striim.md %})]
+
+<!-- CRDB migration service -->
+
+Many other database systems have built-in change data capture (CDC) services that could be used to stream data to CockroachDB while the source database is the source of truth.
+
+After cutover when CockroachDB is the source of truth, [change data capture]({% link {{ page.version.version }}/change-data-capture-overview.md %}) allows you to stream data to the source database.
```

Comment:
obvious comment but suggest putting a big list of migration stuff in a 'See also' heading at the bottom of this